README.md

```diff
@@ -70,11 +70,15 @@ Use Android Studio's Preferences to install NDK (Side by side):
 
 ![Android Studio Preferences Screenshot](docs/images/android-studio-ndk-preferences.png)
 
+Ensuring the correct version:
+![Android Studio Preferences Screenshot](docs/images/android-studio-ndk-version.png)
+
 1. Open Preferences
 2. Navigate to `Appearance & Behavior -> System Settings -> Android SDK` in the sidebar
 3. Navigate to `SDK Tools` tab
-4. Check `NDK (Side by side)` checkbox
-5. Click OK or Apply and follow the download instructions.
+4. Check `Show Package Details` checkbox at the bottom right-hand corner.
+5. Check Version `21.0.6011959` under `NDK (Side by side)`
+6. Click OK or Apply and follow the download instructions.
 
```

Comment:
CMake is was not checked by default. We should instruct the user to check that as well.